docs/guide/using-matlab.md

```diff
@@ -22,7 +22,7 @@ Webots looks for the "matlab" executable in every directory of your *PATH* (or *
 Note that this is similar to calling "matlab" from a terminal (or *Command Prompt* on Windows), therefore, if MATLAB can be started from a terminal then it can also be started from Webots.
 
 On Windows, the MATLAB installer will normally add MATLAB's bin directories to your *Path* environment variable, so usually, Webots will be able to locate MATLAB after a standard installation.
-However, in case it does not work, please make sure that your *Path* environment variable contains this directory (or something slightly different, according to your MATLAB version): `C:\Program Files\MATLAB\R2019b\bin`.
+However, in case it does not work, please make sure that your *Path* environment variable contains this directory (or something slightly different, according to your MATLAB version): `C:\Program Files\MATLAB\R2020b\bin`.
```

Comment:
We should probably use:
```suggestion
However, in case it does not work, please make sure that your *Path* environment variable contains this directory (or something slightly different, according to your MATLAB version): `C:\Program Files\MATLAB\{{ webots.version.major }}\bin`.
```

Reply:
I am hesitating about this as it turns out MATLAB versions often lag behind Webots versions: MATLAB R2020a was released in March 2020 and Webots R2020a was released in December 2019...